tests/register.spec.ts

```diff
@@ -377,18 +380,27 @@ test.describe("when logged in with imported user", () => {
       page.getByRole("heading", { name: `${userFirstName} ${userLastName}` }),
     ).toBeVisible();
     // These are not checked because currently preview doesn't show basic info for imported candidates
-    // TODO: fix these not to be hardcoded
-    /*
+
+    await expect(
+      page.getByText(candidateAppTranslationsEn.languages.Finnish, {
+        exact: true,
+      })
+    ).toBeVisible();
+    await expect(
+      page.getByText(candidateAppTranslationsEn.languages.Spanish, {
+        exact: true,
+      })
+    ).toBeVisible();
+
+
     await expect(
-      page.getByText("Mother Tongues Finnish • Spanish"),
+      page.getByText(candidateAppTranslationsEn.genders.male, {exact: true})
```

Comment:
Instead of translations (because it doesn't necessarily match the one defined for the question), match to `userGender`.